src/Project.php

```diff
@@ -1962,10 +1962,11 @@ public static function getAllKanbanColumns($column_field = null, $column_ids = [
             if (!empty($column_ids) && !$get_default) {
                 $restrict = ['id' => $column_ids];
             }
-            $allstates = $projectstate->find($restrict, ['is_finished ASC', 'id']);
+            $allstates = $projectstate->find($restrict, ['name ASC']);
```

Comment:
If you want to sot by name, you should probably do it in PHP to be able to use a natural sorting method.

Also, I noticed that dropdown translations are not used here. Maybe they should.

Reply:
Last commit do a sort from PHP and handle translation